.github/workflows/main.yml

```diff
@@ -37,3 +37,5 @@ jobs:
       run: make check_pot
     - name: Po file
       run: make check_po
+    - name: Check for missing translations
+      run: make check_missing_translations
```

Comment:
By adding this to the current CI workflow, we trigger an error because of the current 33 missing translations.

We could either:
- Add them within the scope of this PR or;
- Leave the workflow unchanged and add the translations + workflow on another PR

Reply:
They're going to be added after @alexruzenhack PR's are merged